### DISCUSSION TITLE
Possible to get off of verror?

### DISCUSSION THREAD

**Discussion by benasher44** [sequelize/umzug#545] [General] [answered]
@types/verror is preventing us from upgrading TypeScript: https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/59093. This library is the only dependency we have that uses it. Maybe there's a way to avoid exposing its types?

Thanks for your help and consideration!

**Comment by mmkal** [comment, marked as answer]
Yes, I was intending to switch to https://github.com/voxpelli/pony-cause some time soon. A PR doing that would be welcome!

**Reply by mmkal** [reply]
Done! https://github.com/sequelize/umzug/releases/tag/v3.1.0

**Reply by benasher44** [reply]
![](https://media1.tenor.com/images/ea1bbb1a5110e291cc93c14fcedccb2c/tenor.gif?itemid=4488143)
